src/command_modules/azure-cli-relay/azure/cli/command_modules/relay/commands.py

```diff
@@ -0,0 +1,84 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+    from azure.cli.command_modules.relay._client_factory import namespaces_mgmt_client_factory, \
+        wcfrelays_mgmt_client_factory, hycos_mgmt_client_factory
+    from azure.cli.command_modules.relay.custom import empty_on_404
```

Comment:
When you make the changes for the show commands, you'll need to remove this line or pylint will complain.

Reply:
@tjprescott all done. the empty_on_404 is still used by other command so I let that in. The broken build doesn't seem to be my doing. 

I'll be out on vacation for 3 weeks now, so I hope guys can merge and ship.